package.json

```diff
@@ -4,12 +4,12 @@
   "license": "Apache-2.0",
   "main": "dist/girder.umd.min.js",
   "scripts": {
-    "serve": "vue-cli-service serve",
+    "serve": "vue-cli-service serve demo/main.js",
     "build": "vue-cli-service build --target lib --name girder src/index.js",
-    "build:demo": "vue-cli-service build --dest _site/",
-    "lint": "vue-cli-service lint",
-    "lint:pug": "pug-lint-vue src",
-    "lint:style": "stylelint 'src/**/*.vue'",
+    "build:demo": "vue-cli-service build --dest _site/ demo/main.js",
+    "lint": "vue-cli-service lint src/ tests/ demo/ '*.js' '.*.js' '{src,tests,demo}/**/.*.js'",
```

Comment:
Just tested locally and `"vue-cli-service lint src/ tests/ demo/"` seems to work, why are the other arguments necessary?

Reply:
* `'*.js'` checks the root directory
  * for instance, `vue.config.js`
* `'.*.js'` and `'{src,tests,demo}/**/.*.js'"` checks hidden files, which ESLint does not pick up by default
  * for instance, `tests/unit/.eslintrc.js`

This logic comes from https://github.com/vuejs/vue-cli/blob/v3.0.4/packages/@vue/cli-plugin-eslint/lint.js#L18 , I just extended it to add `demo/`.

Reply:
Seems reasonable.